code/index.html

```diff
@@ -3,28 +3,184 @@
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    <header>
+      <div class="video-container">
+      <div id="hero">
+            <video
+            loop
+            muted
+            autoplay
+            preload="auto"
+            src="./videos/Video.mp4"
+            type="video.mp4"
+            type="video/webm">
+            <img src="./images/cyclist.jpg" alt="Front wheel and handlebar of a road cylist">
+        Your broswer does not support this video format.
+      </video>
+      <div class="caption">
+        <p>
+          We take you all the way
+          <span> 
+            It's your Tour! 
+          </span>
+          &mdash; for the road cycling enthusiast 
+          &mdash;
+        </p>
       </div>
+    </header>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <main>
+            
+        <section class="container-1">
+        <div class="form-wrapper">
+        <div class="forms">
+        <div class="login-form">
+            <input type="radio" name="tab" class="tab" id="sign-up-tab" checked />
+            <label for="sign-up-tab" class="tab-header">SIGN UP</label>
+            <input type="radio" name="tab" class="tab" id="sign-in-tab" />
+            <label for="sign-in-tab" class="tab-header">SIGN IN</label>
+            <form id="form1"
+            <form 
+              class="form"
+              action="http://httpbin.org/anything"
+              method="POST"
+              >
+              <div class="header">Sign up for free</div>
+              <div class="form-input">
+                <label for="newUserName">User Name</label>
+                <input
+                  id="newUserName"
+                  type="text"
+                  placeholder="Enter Your User Name"
+                  name="newUserNameValue"
+                  required
+                />
+              </div>
+              <div class="form-input">
+                <label for="email">Email</label>
+                <input
+                  id="email"
+                  type="email"
+                  placeholder="Enter Your Email Address"
+                  name="emailValue"
+                  required
+                />
+              </div>
+              <div class="form-input">
+                <label for="password">Password</label>
+                <input
+                  id="password"
+                  type="password"
+                  placeholder="Enter Password"
+                  name="passwordValue"
+                  required 
+                />
+            <input type="checkbox"/><span id="check">Keep me posted!</span><br />
```

Comment:
We do wanna stay away from using `<br>`since you can tweak these types of things with some CSS instead

Reply:
Thank you for making me aware of that :)